index.js

```diff
@@ -54,7 +68,21 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let average = 0;
+  let rating = 0;
+  if(movies.length < 1){
+    return 0;
+  } 
+    for (let i = 0; i < movies.length; i++){
+      average += Number(movies[i].imdbRating);
+      rating += 1;
+
+    }
+
+    return (average/rating);
+  }
```

Comment:
I like what we have going on with `rating` - however do we need a whole new variable if we have `movies.length?`- aren't they the same thing?  Also lets watch our spacing below 76.